src/cheri_pte.sail

```diff
@@ -49,6 +88,57 @@ union PTE_Check = {
   PTE_Check_Failure : (ext_ptw, ext_ptw_fail)
 }
 
+/*
+ * Assuming we're allowed to load from this page, modulate our cap response
+ */
+val checkPTEPermission_LC : (PTE_Bits, Ext_PTE_Bits, ext_ptw) -> PTE_Check effect { escape, rreg }
+function checkPTEPermission_LC(p, e, ext_ptw) = {
+  let res : PTE_Check = match (e.CapRead(), e.CapRead_Mod(), e.CapRead_Gen()) {
+    (0b0, 0b0, 0b0)  => PTE_Check_Success(ext_ptw_lc_join(ext_ptw, PTW_LC_CLEAR)), /* Clear tag for "unmodified" no-LC case */
+    (0b0, 0b1, 0b0)  => PTE_Check_Success(ext_ptw_lc_join(ext_ptw, PTW_LC_TRAP)),  /* Trap on tag load for "modified" no-LC case */
+    (0b0, _  , 0b1)  => PTE_Check_Failure(ext_ptw, EPTWF_CAP_ERR),
```

Comment:
```suggestion
    (0b0, _  , 0b1)  => PTE_Check_Failure(ext_ptw, EPTWF_INVALID),
```
otherwise this will hit:
https://github.com/CTSRD-CHERI/sail-cheri-riscv/blob/7cfc2d1b148f020e0ab51ddfa8bd7f9a5f23e22f/src/cheri_ptw.sail#L44
(though maybe we want to distinguish these as capability page faults still?)

Reply:
Will we?  I thought we'd hit the `EPTWF_CAP_ERR` case of  `ext_get_ptw_error` and then the `AT_CAP_ERR` cases of `translationException`?

Reply:
Not for `Read`, only `Write` and `ReadWrite` (Peter and I observed this with his in-progress TestRIG translation template).